vendor/engines/secure_rooms/app/views/order_management/order_details/secure_rooms/_occupancy.html.haml

```diff
@@ -1,4 +1,26 @@
-.datetime-block.js--pricingUpdate
+.datetime-block.js--timeFieldAdjustor
   = f.simple_fields_for :occupancy_attributes, f.object.time_data do |of|
     = of.input :entry_at, as: :time_dropdown
+
+    = of.input :actual_duration_mins, input_html: { class: "timeinput" }
+
     = of.input :exit_at, as: :time_dropdown
+
+:coffeescript
```

Comment:
If we weren't in an engine, this would be a little easier to extract out of being inline. If we end up ever adding more JS to the engine, then it might be worth getting it in via a view hook, but that just didn't seem worth it for this, especially as this is a configuration tying directly to the form elements that get rendered in the same partial.